### ISSUE TITLE
Python traceback after clicking apply

### ISSUE THREAD

**Issue by ZechariahB** [realmazharhussain/gdm-settings#130]
### What happened?

Attempting to apply any changes on gdm-settings causes a python error with traceback. Manually compiled and appimage version is affected. Flatpak is not. gdm-settings silently errors without notifying there was an error. In effect, apply does nothing.

```bash
[admin@Lenovo-Legion-ZB Downloads]$ ./Login_Manager_Settings.AppImage -v
/usr/lib/gio/modules/libdconfsettings.so: wrong ELF class: ELFCLASS32
Failed to load module: /usr/lib/gio/modules/libdconfsettings.so
INFO: Application Version    = 1.1
INFO: Operating System       = Fedora Linux 36 (Workstation Edition)
INFO: PackageType            = AppImage
INFO: TEMP_DIR               = /run/user/1000/app/io.github.realmazharhussain.GdmSettings
INFO: HOST_DATA_DIRS         = /usr/local/share:/usr/share
INFO: ShellGresourceFile     = /usr/share/gnome-shell/gnome-shell-theme.gresource
INFO: UbuntuGdmGresourceFile = None
INFO: pkexec version 0.120
INFO: GLib 2.74.0
Traceback (most recent call last):
  File "/tmp/.mount_Login_hV8ykk/usr/lib/python3.10/site-packages/gdm_settings/utils.py", line 130, in <lambda>
    self._finish_callback = lambda self, task, nothing: finish_callback()
  File "/tmp/.mount_Login_hV8ykk/usr/lib/python3.10/site-packages/gdm_settings/window.py", line 121, in on_apply_finished
    if self.apply_task.finish():
  File "/tmp/.mount_Login_hV8ykk/usr/lib/python3.10/site-packages/gdm_settings/utils.py", line 160, in finish
    raise value
  File "/tmp/.mount_Login_hV8ykk/usr/lib/python3.10/site-packages/gdm_settings/utils.py", line 145, in _thread_cb
    retval = self._function()
  File "/tmp/.mount_Login_hV8ykk/usr/lib/python3.10/site-packages/gdm_settings/settings.py", line 398, in apply_settings
    self.apply_shell_theme_settings()
  File "/tmp/.mount_Login_hV8ykk/usr/lib/python3.10/site-packages/gdm_settings/settings.py", line 238, in apply_shell_theme_settings
    compiled_file = gr_utils.compile(shelldir,
  File "/tmp/.mount_Login_hV8ykk/usr/lib/python3.10/site-packages/gdm_settings/gr_utils.py", line 101, in compile
    extract_default_theme(temp_theme_dir)
  File "/tmp/.mount_Login_hV8ykk/usr/lib/python3.10/site-packages/gdm_settings/gr_utils.py", line 67, in extract_default_theme
    resource_list = getstdout(["gresource", "list", env.HOST_ROOT + gresource_file]).decode().splitlines()
TypeError: can only concatenate str (not "NoneType") to str
```

### Expected behavior

Crash should not happen. When a crash does occur, it should be handled with a message that appears notifying there was an error that occurred.

### To Reproduce

Do not use flatpak version. Use appimage or compiled version, then click apply. 

### Application Version

1.1

### Source

Appimage & Source Code

### Environment

- **OS**: Fedora Linux 36 (Workstation Edition)
- **GNOME**: 42.4

**Comment by realmazharhussain** [comment]
Have you tried the latest beta release?

**Comment by ZechariahB** [comment]
I did a boneheaded move and didn't copy my gresource file before reinstalling gnome-shell to compare them. This failure case was wiped clean by that, now I can't identify the root cause of why the python script is not playing well with it. Damn.

**Comment by realmazharhussain** [comment]
This was an issue in v1.0 and had been fixed in v1.1 but a reinstall of gnome-shell is required for the fix to work.

This was mentioned as a note in [release notes for v1.1](https://github.com/realmazharhussain/gdm-settings/releases/tag/v1.1). For more information see <https://github.com/realmazharhussain/gdm-settings/issues/104>.
